packages/worker-activity-page/webpack.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    target: "webworker",
+    resolve: {
+        extensions: ['.ts', '.tsx', '.js', '.json']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/alexwilson/frontend/pullRequest?prid=7165000)
```suggestion
        extensions: [".ts", '.tsx', '.js', '.json']
```